.github/workflows/build_python.yml

```diff
@@ -17,7 +17,7 @@
 # under the License.
 #
 
-name: "Python - PyPy3.8 (master)"
+name: "Build / Python-only (master, PyPy 3.8)"
```

Comment:
BTW, this might be truncated like `Build / Python-only (master....)`. Could you double-check if we can see `3.8`, @HyukjinKwon ?